profiles/getlost01.json

```
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/79409258/227697539-e4aa8792-dae8-49dc-84a1-746028d22192.png",
+    "issueId": 1770,
+    "name": "Aagam Jain",
+    "username": "getlost01"
+  }
+  
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

Changes done